packages/common/src/types.ts

```diff
@@ -345,37 +357,71 @@ export interface TimeParametersV1 {
     mintPerPayday: number;
 }
 
+export interface TimeoutParameters {
+    /**
+     * In milliseconds.
+     */
+    timeoutBase: Duration;
+    timeoutIncrease: Ratio;
+    timeoutDecrease: Ratio;
+}
+
+export interface ConsensusParameters {
+    timeoutParameters: TimeoutParameters;
+    minBlockTime: Duration;
+    blockEnergyLimit: bigint;
+}
+
+export interface FinalizationCommitteeParameters {
+    minimumFinalizers: number;
+    maximumFinalizers: number;
+    finalizerRelativeStakeThreshold: number;
```

Comment:
It's a bit unclear what the "number" represents here. In protobuf schema you get an integer, representing parts per hundred thousand. Is that this integer, or the number `x / 100000`?

Reply:
It is `x / 100000`, which is also the case (and has been since grpcv2 was released) everywhere else the `AmountFraction` type from the GRPC API types is used. Will add a comment when I get to it.